<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>submit_down</class>
 <widget class="QFrame" name="submit_down">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="name_all">
         <item>
          <widget class="QLabel" name="name">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_input">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>111</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="category_all">
         <item>
          <widget class="QLabel" name="category">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>类别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="category_input">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="area_all">
         <item>
          <widget class="QLabel" name="area">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>地区：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="area_input">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="introduction_all">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="introduction">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>31</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>20</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="text">
              <string>简介：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="introduction_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>135</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="history_all">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="history">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>31</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>20</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="text">
              <string>历史：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="history_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>135</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>111</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="image_all">
         <item>
          <widget class="QLabel" name="image">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>图片：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="image_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(172, 172, 172);</string>
           </property>
           <property name="text">
            <string>上传</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="verify_all">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="verify">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>21</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(181, 181, 181);</string>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
